src/index.js

```diff
@@ -1,13 +1,30 @@
 const express = require('express');
 const config = require('config');
 const app = express();
+app.use(express.json());
```

Comment:
could you please let your code breath ?! Add a space between the line 3 and 4 ;-)